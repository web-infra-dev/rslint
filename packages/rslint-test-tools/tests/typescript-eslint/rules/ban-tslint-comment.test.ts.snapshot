exports[`ban-tslint-comment > invalid 1`] = `
{
  "diagnostics": [
    {
      "ruleName": "ban-tslint-comment",
      "messageId": "commentDetected",
      "message": "tslint comment detected: \\"/* tslint:disable */\\"",
      "filePath": "src/virtual.ts",
      "range": {
        "start": {
          "line": 1,
          "column": 1
        },
        "end": {
          "line": 1,
          "column": 21
        }
      }
    }
  ],
  "errorCount": 1,
  "fileCount": 7,
  "ruleCount": 1
}
`;

exports[`ban-tslint-comment > invalid 2`] = `
{
  "diagnostics": [
    {
      "ruleName": "ban-tslint-comment",
      "messageId": "commentDetected",
      "message": "tslint comment detected: \\"/* tslint:enable */\\"",
      "filePath": "src/virtual.ts",
      "range": {
        "start": {
          "line": 1,
          "column": 1
        },
        "end": {
          "line": 1,
          "column": 20
        }
      }
    }
  ],
  "errorCount": 1,
  "fileCount": 7,
  "ruleCount": 1
}
`;

exports[`ban-tslint-comment > invalid 3`] = `
{
  "diagnostics": [
    {
      "ruleName": "ban-tslint-comment",
      "messageId": "commentDetected",
      "message": "tslint comment detected: \\"/* tslint:disable:rule1 rule2 rule3... */\\"",
      "filePath": "src/virtual.ts",
      "range": {
        "start": {
          "line": 1,
          "column": 1
        },
        "end": {
          "line": 1,
          "column": 42
        }
      }
    }
  ],
  "errorCount": 1,
  "fileCount": 7,
  "ruleCount": 1
}
`;

exports[`ban-tslint-comment > invalid 4`] = `
{
  "diagnostics": [
    {
      "ruleName": "ban-tslint-comment",
      "messageId": "commentDetected",
      "message": "tslint comment detected: \\"/* tslint:enable:rule1 rule2 rule3... */\\"",
      "filePath": "src/virtual.ts",
      "range": {
        "start": {
          "line": 1,
          "column": 1
        },
        "end": {
          "line": 1,
          "column": 41
        }
      }
    }
  ],
  "errorCount": 1,
  "fileCount": 7,
  "ruleCount": 1
}
`;

exports[`ban-tslint-comment > invalid 5`] = `
{
  "diagnostics": [
    {
      "ruleName": "ban-tslint-comment",
      "messageId": "commentDetected",
      "message": "tslint comment detected: \\"// tslint:disable-next-line\\"",
      "filePath": "src/virtual.ts",
      "range": {
        "start": {
          "line": 1,
          "column": 1
        },
        "end": {
          "line": 1,
          "column": 28
        }
      }
    }
  ],
  "errorCount": 1,
  "fileCount": 7,
  "ruleCount": 1
}
`;

exports[`ban-tslint-comment > invalid 6`] = `
{
  "diagnostics": [
    {
      "ruleName": "ban-tslint-comment",
      "messageId": "commentDetected",
      "message": "tslint comment detected: \\"// tslint:disable-line\\"",
      "filePath": "src/virtual.ts",
      "range": {
        "start": {
          "line": 1,
          "column": 13
        },
        "end": {
          "line": 1,
          "column": 35
        }
      }
    }
  ],
  "errorCount": 1,
  "fileCount": 7,
  "ruleCount": 1
}
`;

exports[`ban-tslint-comment > invalid 7`] = `
{
  "diagnostics": [
    {
      "ruleName": "ban-tslint-comment",
      "messageId": "commentDetected",
      "message": "tslint comment detected: \\"// tslint:disable-next-line:rule1 rule2 rule3...\\"",
      "filePath": "src/virtual.ts",
      "range": {
        "start": {
          "line": 1,
          "column": 1
        },
        "end": {
          "line": 1,
          "column": 49
        }
      }
    }
  ],
  "errorCount": 1,
  "fileCount": 7,
  "ruleCount": 1
}
`;

exports[`ban-tslint-comment > invalid 8`] = `
{
  "diagnostics": [
    {
      "ruleName": "ban-tslint-comment",
      "messageId": "commentDetected",
      "message": "tslint comment detected: \\"// tslint:disable-line\\"",
      "filePath": "src/virtual.ts",
      "range": {
        "start": {
          "line": 3,
          "column": 1
        },
        "end": {
          "line": 3,
          "column": 23
        }
      }
    }
  ],
  "errorCount": 1,
  "fileCount": 7,
  "ruleCount": 1
}
`;
