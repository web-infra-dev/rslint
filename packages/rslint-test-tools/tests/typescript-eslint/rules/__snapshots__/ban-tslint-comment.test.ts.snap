// Rstest Snapshot v1

exports[`ban-tslint-comment > invalid 1`] = `
{
  "diagnostics": [],
  "errorCount": 0,
  "fileCount": 1,
  "ruleCount": 0,
}
`;
